care/facility/api/serializers/asset.py

```diff
@@ -65,6 +66,16 @@ def validate(self, attrs):
             del attrs["location"]
             attrs["current_location"] = location
 
+        # validate that warraty date is not in the past
+        if "warranty_amc_end_of_validity" in attrs:
+            if attrs["warranty_amc_end_of_validity"] < datetime.now().strftime('%Y-%m-%d'):
```

Comment:
Can you switch this to use actual DateTime comparison, this seems to be a string comparison

Reply:
Done